src/core/manager.c

```diff
@@ -3901,12 +3901,6 @@ char *manager_taint_string(Manager *m) {
         if (r < 0 || !PATH_IN_SET(destination, "../run", "/run"))
                 e = stpcpy(e, "var-run-bad:");
 
-        if (!streq(NOBODY_USER_NAME, "nobody"))
-                e = stpcpy(e, "weird-nobody-user:");
-
-        if (!streq(NOBODY_GROUP_NAME, "nobody"))
-                e = stpcpy(e, "weird-nobody-group:");
-
```

Comment:
hmpf. but ok. but you need to update the size estimation further above too.

Also, could you add a comment here that reminds people that future newly added taints should only be added for non-compile-time things?